analyzers/tests/SonarAnalyzer.Test/Rules/LoggerMembersNamesShouldComplyTest.cs

```diff
@@ -175,4 +175,21 @@ public class Program
             """)
         .AddReferences(NuGetMetadataReference.CastleCore())
         .Verify();
+
+    [TestMethod]
+    public void LoggerMembersNamesShouldComply_Parametrized_CS() =>
+        new VerifierBuilder()
+        .AddAnalyzer(() => new LoggerMembersNamesShouldComply { Format = "^chocolate$" })
+        .AddSnippet("""
+                using System;
```

Comment:
pedantic: indentation is a bit off

Reply:
still is :). Sorry, I was not clear, all the method invocations should be shifted right with 4 spaces.